### DISCUSSION TITLE
Make duplication work across different collections and Flexible Editor instances

### DISCUSSION THREAD

**Discussion by formfcw** [formfcw/directus-extension-flexible-editor#11] [Feature Request]
Duplication (copy&paste, [alt] + drag’n’drop, …) of relation nodes causes some issues. Some parts have to be completely rewritten in order to make duplication work across different collections and Flexible Editor instances. New (edge) use cases will be included (like duplicating an entire content item).

**Comment by jdjoshuadavison** [comment]
When using 'save as a copy', the relation nodes in any flexible editor instance say "The related item does not exist!". We didn't think this would be something we'd do very often, so hadn't considered this a critical issue, but its becoming clear that 'save as a copy' will be a significant part of our workflow, so this issue has become critical for us.

**Comment by jeremyzilar** [comment]
I find the idea of "duplication" confusing in the README. I am not sure how I might use it, thus it makes it hard to know what to set in the `Item Duplication Fields`. It would be great to break out duplication as it's own section and give a little bit of context around the concept as a whole and why it matters to set it up correctly.

**Reply by formfcw** [reply]
Thanks for your thoughts, which I agree with. Duplication is not a topic related to Flexible Editor (alone), it is a Directus topic. Flexible Editor has to deal with it.

Have a look at this feature request, which would simplify the setup of Flexible Editor a lot: https://github.com/directus/directus/discussions/20024
